install/threesdk/cli.py

```diff
@@ -110,15 +110,75 @@ def version():
     Print version number
     """
     print(f"3sdk {_sdk.__version__}")
-    # from InstallTools import get_newest_version
-    dic = _sdk.InstallTools.Tools.get_newest_version()
     # campare with __version__
-    if not dic["up_to_date"]:
+    if not _sdk.InstallTools.Tools.is_latest_release():
+        dic = _sdk.InstallTools.Tools.get_latest_release()
         print(
-            f"Your SDK version is not up-to-date. Newest release is {dic['last_release']}\nPlease visit: {dic['last_release_url']}"
+            f"Your SDK version is not up-to-date. Newest release is {dic['latest_release']}\nPlease visit: {dic['latest_release_url']}\nOr run '3sdk update'"
         )
 
 
+def update():
+    """
+    """
+    from threesdk import container
+    from threesdk.InstallTools import JumpscaleInstaller, MyEnv, Tools, DockerFactory
+
+    if not Tools.is_latest_release():
+        installer = JumpscaleInstaller()
+        containers_names = DockerFactory.containers()
+        host_mount = False
+        # pull all repos in containers
+        for item in containers_names:
+            name = item.name
+            print("Updating repositories in container ", name, "...")
+            container._containers.assert_container(name)
+            c = DockerFactory.container_get(name=name)
+            if c.mount_code_exists:
+                host_mount = True
```

Comment:
you want to keep a list of these containers to loop them later on (after the pull) to restart them (run with latest code)

Reply:
added loop for restart container after checking its type [ simulator - 3bot ] :+1: